modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPAttachmentFileEntryServiceImpl.java

```diff
@@ -60,8 +60,9 @@ public CPAttachmentFileEntry addCPAttachmentFileEntry(
 			int displayDateMinute, int expirationDateMonth,
 			int expirationDateDay, int expirationDateYear,
 			int expirationDateHour, int expirationDateMinute,
-			boolean neverExpire, Map<Locale, String> titleMap, String json,
-			double priority, int type, ServiceContext serviceContext)
+			boolean neverExpire, boolean galleryEnabled,
```

Comment:
why does galleryEnabled come after neverExpire? Shouldn't we sort alphabetically?

Reply:
`neverExpire` is not a real field and it is always bound with the expirationDate. I'll check if there are examples where `neverExpire` doesn't come after `expirationDate`.

Reply:
I've found some in our modules, I'll change it. Thanks!